### ISSUE TITLE
Alert: create variation to use for dynamic notifications

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1421]
The Alert component needs to be extended to support a variation of the Alert that can be used for toast notifications. 

The html that's currently defined in core does not reflect the html needed to support dynamic alerts. Instead, we'd like to base the core example off of what's implemented for this issue.

A [POC](https://github.com/andresgalante/jenn/pull/25) was created to test what works across the different screen readers we support, and findings are noted in this comment: https://github.com/patternfly/patternfly-next/issues/1253#issuecomment-465355653

Given this testing, we found that:
- A live region must exist on a page 200 ms before the alert message is inserted into the live region
- A separate live region is required for each alert

Some suggestions for how to handle the html structure are noted in this comment: https://github.com/patternfly/patternfly-next/issues/1375#issuecomment-465364138

Ultimately, we'd like the react dev team to decide the best way to address these requirements. Then we will update core to match. 

Please ensure that [all links have discernible text](https://dequeuniversity.com/rules/axe/3.4/link-name?application=AxeChrome) and [all id attribute values are unique](https://dequeuniversity.com/rules/axe/3.4/duplicate-id?application=AxeChrome). These are currently known axe issues with alert group in core that should be considered in this implementation. To check if these conditions are met, please run axe on your solution.
